app/build.gradle

```diff
@@ -145,6 +145,7 @@ dependencies {
   androidTestImplementation "com.github.blocoio:faker:$versions.faker"
   androidTestImplementation "android.arch.persistence.room:testing:$versions.room"
   kaptAndroidTest "com.google.dagger:dagger-compiler:$versions.dagger"
+  androidTestImplementation "android.arch.core:core-testing:1.1.1"
```

Comment:
Could you please externalise the version like the other dependencies?

Reply:
Done